src/tds/CMakeLists.txt

```diff
@@ -47,3 +47,12 @@ add_library(tds STATIC
 if (NOT WIN32)
 	set_target_properties(tds PROPERTIES COMPILE_FLAGS -fPIC)
 endif()
+
+if(WIN32)
+INSTALL(TARGETS tds
```

Comment:
not intended to be installed and reused, not sure if you need to compile with static dblib/ctlib